### PR TITLE
Admin CommunityOptionsPopover Carat Fix

### PR DESCRIPTION
**Pull request by alexyoung23j**
Fixes for problems described in [this](https://www.notion.so/hicommonwealth/Add-Admin-Carat-to-all-pages-within-a-community-e7eb65d2ec014fd89f5357f72c1a4dcf) ticket. Note this does not address the product questions posed in the ticket related to choosing between button vs icon. 

## Description

A few minor changes, namely: 
- Carat icon now appears for Proposal and Snapshot Proposal pages as well as Manage Community and Analytics Pages.
- CommunityOptionsPopover modal now centered with carat.
- Mouse hover over carat changes cursor type to indicate expected behavior. 

## Motivation and Context

Admins should have access to this popover from anywhere in the community pages. 

## How has this been tested?

Tested locally for admin and member accounts. 

## Have proper tags been added (for bug, enhancement, breaking change)?
- [ x] yes

## Does this PR affect any server routes?
- [ ] yes, and they are tested: [enter the % coverage here]
- [ ] yes, and they are not tested: [enter the % coverage here]
- [ ] yes, but I did not run tests
- [ x] no

### CODE REVIEW

File: client/scripts/views/pages/proposals.ts
Comment (zakhap) [inline client/scripts/views/pages/proposals.ts:286]: Instead of invoking the CommunityOptionsPopover menu every time there's a sublayout invoked, let's add it to the Sublayout component directly. If you inspect the components in the browser, you can see that they're inside the `.sublayout-header-left` div where the "Breadcrumbs" are defined (using page title). Lines 71 and 85 might be good places to invoke the carat component.

File: client/scripts/views/pages/discussions/index.ts
Comment (zakhap) [inline client/scripts/views/pages/discussions/index.ts:76]: Typically, we don't just write "div" for a component names, we try to use a more "human-readable" class name which can be something like `.dropdown-spacing` or `.dropdown-wrapper`. This will automatically make the mithril component be a div, but it'll be classed something we can quickly understand.

File: client/scripts/views/pages/discussions/index.ts
Comment (zakhap) [inline client/scripts/views/pages/discussions/index.ts:81]: We try to avoid inline styling in almost all cases across the site. The only time we don't is with some tricky table and table cell formatting stuff. This and the css below should be moved into an aptly named scss file and the styles imported appropriately.

File: client/scripts/views/pages/stats.ts
Comment (zakhap) [inline client/scripts/views/pages/stats.ts:142]: Remove this (like discussed above) and other instance in which you see it before the component.

**Review by zakhap — CHANGES_REQUESTED** [review]
Great first PR! A few style things and a few code architecture things, but otherwise looking good! Might need one more review, but it came along well.

File: client/scripts/views/pages/stats.ts
Comment (alexyoung23j) [reply client/scripts/views/pages/stats.ts:142]: Cool! Will fix these things.

File: client/scripts/views/pages/discussions/index.ts
Comment (zakhap) [inline client/scripts/views/pages/discussions/index.ts:88]: is there a reason we can't just add the spacing style directly to the Icon itself by adding a class to the Icon component?

like:
```
m(Icon, {
   name: Icons.CHEVRON_DOWN,
   class: '.dropdown-spacing',
}), // Then we can remove the wrapper and spacer? 
```

File: client/scripts/views/pages/snapshot_proposals/index.ts
Comment (zakhap) [inline client/scripts/views/pages/snapshot_proposals/index.ts:108]: make this just
```
title: 'Proposals',
```
no need for the bracket.

**Review by zakhap — CHANGES_REQUESTED** [review]
Getting really close! Works on all the pages I checked 🔥

File: client/scripts/views/pages/discussions/index.ts
Comment (alexyoung23j) [reply client/scripts/views/pages/discussions/index.ts:88]: I'm not totally sure why but the Icons that are being used all have a ::before pseudo-element in them that's causing a small extra space in the div, which makes the actual SVG offset from the div center. I wasn't able to override the behavior of the ::before with css in the way you described, so I added the extra div to essentially recenter the svg. Played around with this for a while and wasn't able to find a fix, if you think it's important I can keep looking, your call.

Comment (zakhap) [reply client/scripts/views/pages/discussions/index.ts:88]: @gdjohnson do you have any ideas or is this hack fine in your mind?

Comment (gdjohnson) [reply client/scripts/views/pages/discussions/index.ts:88]: It seems fine for now, I can poke into the ::before issue more generally on Monday, but until then this'll do.

**Review by zakhap — APPROVED** [review]
🔥

**Pull request merged**
